install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
ERROR_COUNT, CONVENTION_COUNT, WARNINGS, CODEBASE, THRESHOLD_LINT_SCORE

Above counts/score needs to be modified based on requirement.. Can we add a note on same in readme

Reply:
Added (line:9)